### ISSUE TITLE
[AGENT1 UI HANDOFF] Signal Scan calibrated-probability / 90%-precision integration contract

### ISSUE THREAD

**Issue by copytolive** [copytolive/copy#1]
## Context

Owner split is strict:
- **AGENT4 = Signal Scan SYSTEM / ENGINE / DATA only**
- **AGENT1 = sole UI/frontend/shared integration owner**

AGENT4's previous Wave-2 task is already terminal PASS and currently has `NO_ASSIGNED_TASK`, so **no new Signal Scan source mutation has been made** for this proposal. This issue is a coordination handoff for AGENT1's active final-integration work and for the next AGENT4 engine task.

## Critical finding from read-only screening

Current Signal Hub / scanner outputs do **not** share one statistical meaning for `confidence`.
Examples in existing local implementation include:
- Alpha101: `abs(consensus_score) * 100`
- VectorBT: `abs(bestReturn)` mapped to confidence
- LEAN: absolute return mapped to confidence
- HFT: return transformed into confidence
- PyBroker: average accuracy used as current-signal confidence
- final consensus: simple LONG-vs-SHORT engine count

Therefore **`confidence: 90` MUST NOT be rendered or described as `90% probability of winning`**.

## New target semantics

The product target should be **high selective precision**, not cosmetic confidence:

> Among signals that pass all calibration / OOS / uncertainty gates and are actually emitted as tradable candidates, target precision may be evaluated toward ~90%.

This is an R&D target, **not a guaranteed win rate**.

## Proposed future AGENT4 nonvisual contract

Future Signal Scan engine work should expose evidence similar to:

```ts
interface CalibratedSignalEvidence {
  rawConfidence: number | null;
  calibratedPWin: number | null;
  calibrationMethod: 'platt' | 'isotonic' | 'conformal' | null;
  uncertainty: number | null;
  decision: 'TAKE' | 'ABSTAIN' | 'REJECT';
  decisionReason: string;
  oosPrecision: number | null;
  oosSampleSize: number | null;
  brierScore: number | null;
  expectedCalibrationError: number | null;
  regime: string | null;
  pair: string;
  timeframe: string;
  evidenceState: 'unavailable' | 'insufficient_sample' | 'calibrating' | 'validated' | 'drifted';
}
```

Exact field names may change when AGENT4 receives a controller task; AGENT1 should not invent/fake them before then.

## AGENT1 UI requirements NOW

1. Keep consuming the current canonical Signal Scan handoff only.
2. **Do not relabel raw `confidence` as probability / win probability.**
3. If the UI shows current confidence, label it explicitly as **engine confidence / score**.
4. Keep `loading`, `empty`, `error`, `reconnect`, `stale`, and `unavailable` visually distinct.
5. Never fabricate calibration values, probability, sample size, OOS precision, Brier score, or uncertainty.
6. When the future calibrated contract lands, add these states:
   - `VALIDATED`
   - `CALIBRATING`
   - `INSUFFICIENT SAMPLE`
   - `DRIFTED`
   - `ABSTAIN / NO TRADE`
7. `ABSTAIN / NO TRADE` is a valid successful output, not an error.
8. UI should prefer evidence labels such as:
   - `Calibrated P(win)`
   - `OOS precision`
   - `OOS sample`
   - `Uncertainty`
   - `Regime`
   - `Decision`
   rather than a single oversized confidence percentage.

## Future AGENT4 implementation priorities

P0:
1. canonical probability contract
2. outcome labeler (e.g. TP-before-SL within fixed horizon)
3. Platt / isotonic calibration
4. conformal uncertainty / abstention layer
5. leakage gate
6. purged chronological walk-forward/OOS validation
7. meta-labeler
8. learned ensemble weights
9. engine-correlation guard
10. minimum-evidence gate

P1:
- pair/timeframe/regime-specific calibration
- probability ledger
- precision/recall/Brier/ECE metrics
- concept-drift detector
- champion/challenger model flow
- multiple-testing / overfit guard

## GitHub components selected for acceleration

- `scikit-learn-contrib/MAPIE`: conformal prediction / uncertainty / risk-control methodology
- `freqtrade/freqtrade`: lookahead-analysis and recursive-analysis concepts
- `microsoft/qlib`: point-in-time / online rolling / concept-drift architecture (already partly integrated locally; do not duplicate Qlib)
- `optuna/optuna`: threshold / ensemble-weight / calibration optimization, train-validation only
- `polakowo/vectorbt`: robustness / walk-forward / massive parameter validation (already partly integrated locally; do not treat return as probability)

## Integration rule

AGENT1 must **not** mutate Signal Scan engine/system/data. AGENT4 must **not** mutate UI/frontend. The final UI should consume only a stable nonvisual contract after AGENT4 receives a new controller task and that task reaches PASS.

## Current coordination status

- AGENT4 previous task: terminal PASS
- AGENT4 current task inbox: `NO_ASSIGNED_TASK`
- AGENT1 current task: `ONEUI-FINAL-A1-INTEGRATION-20260909` assigned
- This issue is coordination/evidence only; it does not claim the calibrated-probability engine already exists.

**Comment by copytolive** [comment]
Synchronization update: this UI handoff is now governed by **#2 — [MASTER SYNC] Calibrated Probability → Execution Preflight → UI Integration → QA**.

Current controller split confirmed locally:
- A8: `ONEUI-W5-A8-PROBABILITY-VALIDATION-20260909`
- A7: `ONEUI-W5-A7-EXECUTION-PREFLIGHT-20260909`
- A1: `ONEUI-FINAL-A1-INTEGRATION-20260909`
- A4: `NO_ASSIGNED_TASK` (must not duplicate A8 calibration work)
- A10: `NO_ASSIGNED_TASK` (future independent QA only)

AGENT1 should consume the **final A8/A7 INTEGRATION_HANDOFF contracts after their terminal PASS**, and use this issue only for UI semantics. Do not implement probability math in frontend.

**Comment by copytolive** [comment]
## Synchronization update — use Master Sync #2 as the cross-agent authority

For AGENT1 integration, this earlier Signal Scan UI handoff is now subordinate to **#2 `[MASTER SYNC] Calibrated Probability → Execution Preflight → UI Integration → QA`** for the new probability/execution chain.

AGENT1 should preserve the useful rule from this issue (`raw confidence != P(win)`), but MUST consume the final cross-agent flow from #2:

`AGENT4 raw scanner evidence → AGENT8 calibrated probability authorization → AGENT7 execution preflight → AGENT1 UI/app → AGENT10 QA`.

Do not implement speculative field names from this older planning issue if A8/A7 final handoffs use different exact exports.

Consumption requires BOTH:
1. terminal PASS for the exact upstream TASK_ID, and
2. final upstream `AI/INTEGRATION_HANDOFF.md` with exact exports.

Until then, unavailable/new probability/preflight values remain unavailable; never fabricate them.
